.github/.wordlist.txt

```diff
@@ -73,6 +73,7 @@ argc
 args
 argv
 armeabi
+armgcc
```

Comment:
can we fix up quoting in the md files to not add all these as `words`?

Reply:
Reviewed changes to reduce list of new words